src/plugins/add-cal/_base.scss

```diff
@@ -0,0 +1,29 @@
+/*
+ * Web Experience Toolkit (WET) / Boîte à outils de l'expérience Web (BOEW)
+ * wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+ */
+
+.wb-addcal.provisional {
```

Comment:
Do we need all that CSS? Can we simply change the CSS class we are adding when inserting the "detail/summary" elements?

If we start to override default Bootstrap CSS (ex ".btn-link"), then we should avoid to use then or see if we should create an generic helper instead.

Reply:
> Do we need all that CSS? Can we simply change the CSS class we are adding when inserting the "detail/summary" elements?
> 
> If we start to override default Bootstrap CSS (ex ".btn-link"), then we should avoid to use then or see if we should create an generic helper instead.

Would have been easier if the two elements were the same (two buttons or two anchors). But in this case, the code was developed to have an <a> tag and a <button> tag. To make them look them same, I either have to override all the default styles of buttons to make them look like anchors and style the anchor the same, or add the class ".btn-link" to both and add a few styles to correct it. I chose the latter.

The other option would have been to have two buttons and the Google one acting as an anchor, but I preferred keeping the default behavior of the anchor since it was possible.

Reply:
Or can the plugin be update to use a template?